src/apps/vpn/controller.cpp

```diff
@@ -519,6 +520,8 @@ void Controller::connected(const QString& pubkey,
     resetConnectedTime();
   }
 
+  mozilla::glean::settings::dns_type.set(DNSHelper::getDNSType());
```

Comment:
Not here. But in telemetry.cpp

Reply:
Per discussion a couple days ago, I believe this is appropriate. We're going to reserve `telemetry.cpp` for complex telemetry recordings, multiline ones, etc.